Misc changes for network section (#678)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_retrieving-network-information-from-your-existing-deployment.adoc

* A list of isolated networks that are used in the existing deployment.
* For each of the isolated networks, the VLAN tag and IP ranges used for
  dynamic address allocation.
* A list of existing IP address allocations that are used in the environment.

>   When reusing the existing subnet ranges to host the new control plane
>   services, these addresses will be excluded from the corresponding allocation
>   pools.

Reviewer: ```suggestion
  When reusing the existing subnet ranges to host the new control plane
  services, these addresses are excluded from the corresponding allocation
  pools.
```

      allocation_pools: [{'start': '172.17.0.4', 'end': '172.17.0.250'}]
----

. Retrieve the VLAN tag that is used in the `vlan` key and the IP range in the
  `ip_subnet` key for each isolated network from the `network_data.yaml` file.
  When reusing subnet ranges from the existing deployment for the new control

>   plane services, the ranges will be split into separate pools for control
>   plane services and load balancer IP addresses.

Reviewer: ```suggestion
  plane services, the ranges are split into separate pools for control
  plane services and load-balancer IP addresses.
```
